src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +27,406 @@ export const AboutApp: React.FC = () => {
+                mb={2}
+              >
+                Partnership Overview
+              </Button>
+              <Button
+                bg="transparent"
+                width="100%"
+                border="none"
+                // @ts-ignore
+                as={RouterLink}
+                to="/galleries"
+              >
+                See Full List of Partners
+              </Button>
             </Column>
           </GridColumns>
-        </Box>
+        </HorizontalPadding>
+      </AppContainer>
+    </Container>
+  )
+}
 
-        <Box py={20} mt={2} data-test="about3">
-          <GridColumns>
-            <Column span={4}>
-              <Text variant="xs">The Artsy iPhone App</Text>
-              <Text variant="lg">Access from anywhere</Text>
-              <Text variant="md">
-                Our iPhone app allows you to browse, save, learn about, and
-                collect from your phone. It also serves as a personalized,
-                on-the-ground guide to any art fair we feature on Artsy.
-              </Text>
-            </Column>
-            <Column span={8}>
-              <Box py={2}>
-                <Image
-                  width="720px"
-                  height="507px"
-                  src="https://artsy-media-uploads.s3.amazonaws.com/PiE_1gCyYvs3IKWiWjHo3g%2FFrame+14.png"
-                />
-              </Box>
-            </Column>
-          </GridColumns>
-        </Box>
-      </Box>
+const AppDownloadSection: React.FC = () => {
+  const image = resized("http://files.artsy.net/download_artsy_apps_img.jpg", {
+    width: 910,
+    height: 652,
+  })
 
-      <Box id="collecting">
-        <Box py={20} data-test="collecting1">
-          <Text variant="xl">COLLECTING</Text>
-          <Text variant="md">
-            There are currently over 300,000 works available for sale on Artsy,
-            provided by galleries, museums, and institutions from around the
-            world. These works range in price from $100 to over $1,000,000, with
-            new ones added every day. Artsy takes no commission from the sale of
-            artworks from our subscribing gallery partners. We serve as a link
-            between our gallery partners and those interested in collecting.
-          </Text>
-        </Box>
+  return (
+    <GridColumns>
+      <Column span={6} mt={6}>
+        <Text variant="xs" textTransform="uppercase">
+          Artsy App
+        </Text>
+        <Text variant="xl" mt={1}>
+          The Art Market at Your Fingertips
+        </Text>
+        <Text variant="sm" mt={1} mb={4}>
+          Discover artworks just for you, get market insights, and buy and sell
+          with confidence—all on the Artsy app.
+        </Text>
+        <Flex>
+          <RouterLink to="https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080">
+            <Image
+              src="http://files.artsy.net/images/download-ios-app-transparent.svg"
```

Comment:
Yeah it's totally fine to use SVGs like this. Whenever we inline them we're including them in the code bundle and sending them over the wire with the HTML page response. This is frequently desirable (like in the case of Palette icons) because it lets us control their fill color. In the case of a one-off like this, just using it like an image is great. One of the benefits is that the browser can cache them like normal images (in this case Vanity doesn't send the correct cache-control headers which... is a separate problem). And ideally we'd be able to also use `lazyLoad` here, which we can't because Palette's image component makes assumptions about the bg color (another separate problem).